include/braincloud/BrainCloudAppStore.h

```diff
@@ -125,6 +125,14 @@ namespace BrainCloud
         */
         void finalizePurchase(const std::string& in_storeId, const std::string& in_transactionId, const std::string& in_jsonTransactionData, IServerCallback* in_callback = NULL);
 
+        /**
```

Comment:
Not sure if this is a big deal, but this is a copied comment from the finalizePurchase function and perhaps should be deleted or edited to relate to the function.

Reply:
Good catch. yes big deal-ish